src/components/Footer.js

```diff
@@ -0,0 +1,39 @@
+import StyledFooter from './styled-components/StyledFooter';
+import StyledImage from './styled-components/StyledImage';
+import StyledUL from './styled-components/StyledUL';
+
+export default function AddProject1() {
```

Comment:
As you use \<Footer /> in your Layout.js, I guess, this export should be called Footer to work. 🤔